Zend/zend_API.h

```diff
@@ -1427,10 +1442,10 @@ ZEND_API ZEND_COLD void ZEND_FASTCALL zend_argument_value_error(uint32_t arg_num
 		if (UNEXPECTED(!zend_parse_arg_object(_arg, &dest, _ce, check_null))) { \
 			if (_ce) { \
 				_error = ZSTR_VAL((_ce)->name); \
-				_error_code = ZPP_ERROR_WRONG_CLASS; \
+				_error_code = check_null ? ZPP_ERROR_WRONG_CLASS_OR_NULL : ZPP_ERROR_WRONG_CLASS; \
```

Comment:
Unfortuantely, I didn't manage to concat the class name with `?` again (e.g. `zend_spprintf()` gave a memory leak), and this was my second best idea. :/

Reply:
@nikic Do you have objection? If not, should the change be documented in the internals upgrading file?

Reply:
Looks reasonable. This is an implementation detail, so no need for upgrading notes.